packages/scope-sdk/src/Scope.ts

```diff
@@ -166,6 +176,11 @@ export class Scope {
     { id: 39, pair: 'UXDTwap/USD', name: 'UXDTwap', price: new Decimal(0) },
     { id: 40, pair: 'HDG/USD', name: 'HDG', price: new Decimal(0) },
     { id: 41, pair: 'DUST/USD', name: 'DUST', price: new Decimal(0) },
+    { id: 42, pair: 'kUSDHUSDCOrca/USD', name: 'kUSDHUSDCOrca/USD', price: new Decimal(0) },
+    { id: 43, pair: 'kSOLSTSOLOrca/USD', name: 'kSOLSTSOLOrca/USD', price: new Decimal(0) },
+    { id: 44, pair: 'kUSDCUSDTOrca/USD', name: 'kUSDCUSDTOrca/USD', price: new Decimal(0) },
+    { id: 45, pair: 'kUSHUSDCOrca/USD', name: 'kUSHUSDCOrca/USD', price: new Decimal(0) },
```

Comment:
```suggestion
    { id: 42, pair: 'kUSDHUSDCOrca/USD', name: 'kUSDHUSDCOrca', price: new Decimal(0) },
    { id: 43, pair: 'kSOLSTSOLOrca/USD', name: 'kSOLSTSOLOrca', price: new Decimal(0) },
    { id: 44, pair: 'kUSDCUSDTOrca/USD', name: 'kUSDCUSDTOrca', price: new Decimal(0) },
    { id: 45, pair: 'kUSHUSDCOrca/USD', name: 'kUSHUSDCOrca', price: new Decimal(0) },
```

Same, names don't have the quote.